src/main/java/de/tum/in/www1/artemis/service/ExamService.java

```diff
@@ -265,11 +328,6 @@ public void delete(Long examId) {
 
             studentExams.add(studentExam);
         }
-
-        studentExams = studentExamRepository.saveAll(studentExams);
```

Comment:
Is it intended that we don't save anymore? If so, could you maybe document that the caller has to save the returned exams?

Reply:
The caller saves them. We can also save the directly there.